docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,60 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged
+   into `develop`.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
+      recipe versions. The script uses build manifests - lists of recipes
+      contained in the image along with their versions. You must provide your
+      manifest and the current release's manifest as arguments to the script.
+      The latter can be found in the latest release's section in the GitHub
+      release pages. The current manifest can be found under the Yocto working
+      directory in
+      `build/tmp/deploy/images/{machine-name}/{image-name}.manifest`.
+    - Paste the script output into your release's section in `CHANGELOG.md`.
+      Additionally, you must explain why those changes were made and what they
+      introduce (unless the commit messages listed by `generate-changelog.sh`
+      already explain that).
+    - Describe any additional changes.
+4. Commit and push the changelog.
+5. Merge the changes from `develop` into `main`.
+5. Create and push tag that matches the newly bumped version with `v` added at
+   the beginning (e.g. `v2.0.1-rc1`).
+6. Publish the release on GitHub. Make sure to upload the manifest. In case of a
```

Comment:
Do we have any process how to add a release (RC release as well) into GitHub page (like what structure to use, etc.)? @PLangowski , could you add more details here?

Reply:
cccce5c6d23977b4f5e7045fa772a1f1e6131056

Reply:
> The release description should be the same as the release notes for that version

To what notes you are referring to?

Reply:
> In case of a `release candidate`, you should publish it as a pre-release.

What does it mean to publish as a pre-release, some specific steps?

Reply:
![image](https://github.com/user-attachments/assets/36023ed8-ffa4-4208-a34a-8fe0a81d850e)

Reply:
@DaniilKl I meant the changelog's release notes which are mentioned a few steps before.

Reply:
And changelog should be review and cleaned up if necessary. There could be multiple `bump srcrev to` for the same recipe but only one (current one) should be mentioned

Reply:
:thinking:, could it be done automatically by the script?

Reply:
@DaniilKl Maybe some day. For now I think it's enough that the person responsible for the release reviews the change notes generated by `git cliff`.

Reply:
I doubt that at least not in generic way. Current one doesn't even know anything about Yocto, I think it just checks `git log`

Reply:
If everything is ok for now, then let's merge this